### ISSUE TITLE
Remove mass size in kilograms and described size from the Mass slider focus alert

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#128]
From https://github.com/phetsims/gravity-force-lab/issues/126#issuecomment-447378450

Please remove the following from the Mass Focus Alert
1. The actual size in kilograms repeats in the alert. Please remove it from the alert. It should only be provided by the slider (i.e., in the aria-valuetext).

2. Based on feedback from screen reader consultants, I've decided the described size of the spheres (e.g., tiny, very small, small, etc) provided in the focus alert, is not that useful, and really not useful in GFL:B. Please remove the described size information (i.e., {{mass-size-qualitative}}) from the focus alert. I've crossed this column out in the design and will remove it soon.

**Comment by zepumph** [comment]
Discussed with @terracoda tonight (this morning) and we think this may be already fixed. Assigning to @terracoda to verify.

**Comment by terracoda** [comment]
@zepumph, I verified that the described mass size information (i.e., {{mass-size-qualitative}} has been removed from the alert.

Also, to note, the implementation of the Mass sliders is now using aria-valuetext (like the position sliders) to provide the mass description information (mass of sphere in kilograms and a qualitative comparative size description). This is correct.

Thanks @mbarlow12 for getting this implemented. You've done awesome work on this sim! 

Closing this issue.
